### PR TITLE
ability to disable certificate check

### PR DESCRIPTION
**Pull request by theromis**

### CODE REVIEW

**Comment by loeffel-io** [comment]
@theromis can you please take care about the conflicts?

**Comment by theromis** [comment]
@loeffel-io done, please check

**Comment by theromis** [comment]
@loeffel-io I don't think I'm exporting something in this commit. Could you clarify your request?

I have a chat app, and sometimes dns is not resolvable, so connecting over HTTPS fails in case of IP connection. This is why disabling ssl cert is critical for me.

**Comment by loeffel-io** [comment]
@theromis your `TLSClientConfig` property is exported

**Comment by theromis** [comment]
@loeffel-io do you want me to close PR?

File: recws.go [hunk 1]
```diff
@@ -36,6 +37,7 @@ type RecConn struct {
 	// Proxy specifies the proxy function for the dialer
 	// defaults to ProxyFromEnvironment
 	Proxy func(*http.Request) (*url.URL, error)
+	TLSClientConfig  *tls.Config
```
Comment (loeffel-io) [inline recws.go]: Please just add a comment to this property as all other properties

Comment (theromis) [reply recws.go]: done, please check

**Pull request merged**